src/entrypoints/options/components/config-card.tsx

```diff
@@ -1,13 +1,26 @@
-export function ConfigCard({ title, description, children, className }: { title: React.ReactNode, description: React.ReactNode, children: React.ReactNode, className?: string }) {
+export function ConfigCard({
```

Comment:
the eslint seems not working in your ide, please fix the style. find out why eslint does not work

the expected behavior is every time you commit, the eslint will format your code automatically.